applications/stepswitch/src/stepswitch_bridge.erl

```diff
@@ -578,7 +579,7 @@ is_emergency_endpoint('true') ->
     'true';
 is_emergency_endpoint('false') -> 'false';
```

Comment:
I totally understand this is not the code that's updated in this PR. I raise this because I can see you try to improve the code quality. So why not they just do
`is_emergency_endpoint(B, _Endpoint) when is_boolean(B) -> B;`
Is it because they need to log? The other issue is that one variable used for both is_emergency and endpoint are confusing.  I hope there should be two parameters from the beginning. What do you think?